docs/docs/reference/cloud/monorepos.md

```diff
@@ -29,7 +29,7 @@ The following table shows the level of support for each tool.
 | Yarn Workspaces (v2/v3 with PnP) | ◐                |                                                         |
 | Lerna                            | ●                |                                                         |
 | NPM Workspaces (v7 and onwards)  | ●                |                                                         |
-| NX                               | ○                | Work in progress                                        |
+| NX                               | ●                | Full support for detecting and installing dependencies   |
```

Comment:
@wardpeet is it fair to call this full support?

Reply:
Based on the feature PR I think it is fair to call it Full support for detecting and installing dependencies